Xamarin.Essentials/AppInfo/AppInfo.android.cs

```diff
@@ -48,5 +51,16 @@ static void PlatformShowSettingsUI()
             settingsIntent.AddFlags(ActivityFlags.ExcludeFromRecents);
             context.StartActivity(settingsIntent);
         }
+
+        static WindowSize PlatformWindowAppSize()
+        {
+            var context = Platform.GetCurrentActivity(false) ?? Platform.AppContext;
+            var windowManager = context.GetSystemService(Context.WindowService);
+            var windows = windowManager.JavaCast<IWindowManager>();
```

Comment:
needs using statements here. We also have helpers that get a lot of this information in displayinfo, may want to see if we have some re-use there.

Reply:
I can use the methods in `DeviceDisplay.Android.cs`, tks for the tip.

Reply:
@jamesmontemagno, I realized if I use the method `DeviceDisplay.GetDefaultDisplay()` it will return the wrong value (returns the screen size). I think the reason it's because the method uses the `ApplicationContext` and to get the app's window a need to use the `CurrentActivity`.